packages/components/src/Dropzone/sub-components/Hyperlink/Hyperlink.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import classNames from "classnames"
+
+interface HyperlinkProps {
+  text: string,
+  href?: `https://${string}`
+  openNewTab?: boolean,
+  handleOnClick?: () => void
+}
+
+export const Hyperlink = ({
```

Comment:
Because this component's styles are strictly for Dropzone, we recommend calling it something very specific like `SelectFileButton`

Reply:
Great call out, and I have refactored the component to only do one thing instead of account for the 'link'. Which was mentioned in this comment here: https://github.com/cultureamp/kaizen-design-system/pull/3353#discussion_r1136501150